docs/uninstall.md

```diff
@@ -99,6 +100,14 @@ The recommended steps to uninstall the OpenEBS cluster gracefully is as follows.
   ```
 
 
+- Ensure that there are no stale BlockDevices present in the cluster. You can verify the status using the following command. 
+  
+  ```
+  kubectl get bd -n <openebs namespace>
+  ```
+  
+  If present, remove the finalizer entry from the corresponding BD.
```

Comment:
If present, remove the finalizer entry from the corresponding BD -> If present, remove the finalizer entry from the corresponding BD and then delete it.